go/kbfs/libfs/fs.go

```diff
@@ -569,8 +569,10 @@ func (fs *FS) ensureParentDir(filename string) error {
 	return nil
 }
 
-func (fs *FS) requireNonEmpty() error {
-	if fs.empty {
+func (fs *FS) checkEmpty(notExistErrIfEmpty bool) error {
```

Comment:
I'd just call this field `errIfEmpty`.

Reply:
it chooses which error to return, not whether to return one at all.